universe/base.go

```diff
@@ -241,24 +241,26 @@ func (a *MintingArchive) FetchIssuanceProof(ctx context.Context, id Identifier,
 				context.Background(), id, key,
 			)
 			if err != nil {
-				log.Warnf("unable to log sync event "+
-					"(id=%v, key=%v): %v", spew.Sdump(id),
+				log.Warnf("unable to log sync event (id=%v, "+
+					"key=%v): %v", id.StringForLog(),
 					spew.Sdump(key), err)
 			}
 		}()
 	}()
 
-	return withBaseUni(a, id, func(baseUni BaseBackend) ([]*IssuanceProof, error) {
-		return baseUni.FetchIssuanceProof(ctx, key)
-	})
+	return withBaseUni(
```

Comment:
Line was too long. I guess it should go to its own commit, but since we only touch this file once in this PR I thought it was fine to sneak some formatting changes in.